classes/controller/Controller.php

```diff
@@ -346,7 +346,15 @@ public function run()
 
     protected function trans($id, array $parameters = [], $domain = null, $locale = null)
     {
-        $parameters['legacy'] = 'htmlspecialchars';
+        if (!isset($parameters['_raw'])) {
+            return htmlspecialchars($this->translator->trans($id, $parameters, $domain, $locale), ENT_NOQUOTES);
+        }
+
+        @trigger_error(
+            'Use _raw parameter will be removed in version 9.0, escaping translation is not the translator responsability.',
+            E_USER_DEPRECATED
+        );
+        unset($parameters['_raw']);
 
         return $this->translator->trans($id, $parameters, $domain, $locale);
```

Comment:
Same change as in the other function:

```suggestion
        if (isset($parameters['_raw'])) {
            @trigger_error(
                'The _raw parameter is deprecated and will be removed in the next major version.',
                E_USER_DEPRECATED
            );
            unset($parameters['_raw']);
            
            return $this->translator->trans($id, $parameters, $domain, $locale);
        }
        
        return htmlspecialchars($this->translator->trans($id, $parameters, $domain, $locale), ENT_NOQUOTES);
```